### PR TITLE
chore(client): new UI for the tier-card

### PR DESCRIPTION
**Pull request by Abhishek332**
### **Old UI**
Screenshot 2023-07-21 at 12 04 15 PM

### **New UI**
Screenshot 2023-07-21 at 4 56 51 PM

https://github.com/open-metadata/OpenMetadata/assets/80886271/4031ba18-9627-4a2b-9d2d-38b0737be5fb

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/TierCard/tier-card.style.less [hunk 1]
```diff
@@ -27,34 +36,38 @@
...
+  //       }
+  //     }
+  //     &:first-child,
+  //     &:first-child.ant-collapse-item-active {
+  //       border-radius: 4px 4px 0 0;
+  //     }
+  //     &:last-child {
+  //       border-radius: 0 0 4px 4px;
+  //     }
+  //   }
```
Comment (Ashish8689) [inline openmetadata-ui/src/main/resources/ui/src/components/common/TierCard/tier-card.style.less:70]: remove this commented code.

Comment (Abhishek332) [reply openmetadata-ui/src/main/resources/ui/src/components/common/TierCard/tier-card.style.less]: done

**Review by karanh37 — CHANGES_REQUESTED** [review]
The UI is not matching the mocks. I have highlighted few points. Please take care of it.

**Review by karanh37 — APPROVED** [review]
LGTM

**Comment by karanh37** [comment]
@harshach - can you review the new Tier changes ?

**Review by Ashish8689 — APPROVED** [review]

**Review by harshach — APPROVED** [review]

**Pull request merged**